app/uk/gov/hmrc/bindingtariffclassification/model/MongoFormatters.scala

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.bindingtariffclassification.model
+
+import java.time.{Instant, ZoneId, ZonedDateTime}
+
+import play.api.libs.json._
+import uk.gov.hmrc.play.json.Union
+
+object MongoFormatters {
+
+  implicit val instantFormat: OFormat[ZonedDateTime] = new OFormat[ZonedDateTime] {
```

Comment:
If you use Joda Time, I believe you wouldn't need to define these custom formatters.
There are in fact implicit date/time formatters already available in the Play reactive library. 
I have used them.

I do not know if we have those for the `java.time` objects.
You could ask in the #mongo-community Slack channel in case

Reply:
The implicit formatters format dates as Strings, rather than how Mongo needs them. I cant imagine Joda time would be any different

Reply:
@ed0906 
Can you try using `ReactiveMongoFormats.dateTimeFormats`? 
See these examples:
https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/models/JsonFormatters.scala#L149
https://github.com/hmrc/third-party-delegated-authority/blob/master/app/uk/gov/hmrc/models/JsonFormatters.scala#L71

Reply:
I'm reluctant to use joda time, its something we should be getting rid of now Java 8 is released. I think there needs to be an update to the ReactiveMongoFormats if we want to use them

Reply:
I meant: can you try to see if `ReactiveMongoFormats.dateTimeFormats` works also with Java 8 time types?
Maybe it does.